src/catalogs/cesiumCatalog.ts

```diff
@@ -1,5 +1,9 @@
 import type {INGVCatalog} from '../interfaces/cesium/ingv-catalog.js';
 
+export const ION_ASSETS_URL = 'https://assets.ion.cesium.com/';
```

Comment:
Should be private.

Reply:
it can't be private because it is used outside, I thought to move it to cesium utils